drivers/SmartThings/matter-media/src/init.lua

```diff
@@ -207,6 +212,7 @@ local matter_driver_template = {
       },
       [clusters.MediaPlayback.ID] = {
         [clusters.MediaPlayback.attributes.CurrentState.ID] = media_playback_state_attr_handler,
+        --TODO [clusters.MediaPlayback.attributes.PlaybackSpeed.ID] = media_playback_speed_attr_handler,
```

Comment:
I intend to add this functionality in this PR

Reply:
Opted for just calling out that Advanced Seek features are not supported. It ended up being quite complicated.